src/extensions/default/CodeSnippetsAdder/main.js

```diff
@@ -0,0 +1,34 @@
+/*jslint vars: true, plusplus: true, devel: true, nomen: true, regexp: true, indent: 4, maxerr: 50 */
+/*global define, brackets */
+
+define(function (require, exports, module) {
+    "use strict";
+
+    var CommandManager = brackets.getModule("command/CommandManager");
+	var EditorManager  = brackets.getModule("editor/EditorManager");
```

Comment:
Watch your indentation here, it should line up with what you have at line 7 (i.e., 4 spaces)